en/docs/learn/export-user-info-to-mailChimp.md

```diff
@@ -0,0 +1,30 @@
+#Exporting WSO2 Identity Server User Information to MailChimp
+
+
+MailChimp is an all‑in‑one Marketing Platform. 
+This topic provides instructions on exporting user data from WSO2 Identity Server (WSO2 IS) to the 
+MailChimp. 
+
+
+Let's get started!
+
+!!! tip "Before you begin!"
+    Create **users.csv** file by following the instructions in 
+    [Exporting User Data From WSO2 Identity Server](../learn/exporting-user-data-from-wso2-is.md).
```

Comment:
(../learn/exporting-user-data-from-wso2-is.md) --> (../learn/exporting-user-data-from-wso2-is)

(remove the .md from the link)